src/windows/windows-host-session.vala

```diff
@@ -98,12 +98,17 @@ namespace Frida {
 			var blob64 = Frida.Data.Agent.get_frida_agent_64_dll_blob ();
 			var dbghelp32 = Frida.Data.Agent.get_dbghelp_32_dll_blob ();
 			var dbghelp64 = Frida.Data.Agent.get_dbghelp_64_dll_blob ();
+			var symsrv32 = Frida.Data.Agent.get_symsrv_32_dll_blob ();
+			var symsrv64 = Frida.Data.Agent.get_symsrv_64_dll_blob ();
+
 			agent_desc = new AgentDescriptor.with_resources ("frida-agent-%u.dll",
 				new MemoryInputStream.from_data (blob32.data, null),
 				new MemoryInputStream.from_data (blob64.data, null),
 				new AgentResource[] {
-					new AgentResource ("dbghelp-32.dll", new MemoryInputStream.from_data (dbghelp32.data, null)),
-					new AgentResource ("dbghelp-64.dll", new MemoryInputStream.from_data (dbghelp64.data, null))
+					new AgentResource ("32\\dbghelp.dll", new MemoryInputStream.from_data (dbghelp32.data, null)),
+					new AgentResource ("32\\symsrv.dll", new MemoryInputStream.from_data (symsrv32.data, null)),
+					new AgentResource ("64\\dbghelp.dll", new MemoryInputStream.from_data (dbghelp64.data, null)),
+					new AgentResource ("64\\symsrv.dll", new MemoryInputStream.from_data (symsrv64.data, null))

```

Comment:
I suppose we should also move `frida-agent.dll` to the same directory so `dbghelp.dll` is next to it?

Reply:
Current code just worked fine with frida-agent.dll in parent directory of dbghelp.dll but it would be nicer. Working on it.

Reply:
Fixed.